sormas-api/src/main/resources/strings.properties

```diff
@@ -558,12 +563,17 @@ infoUserEmail = Used to send E-Mail notifications.
 infoMergingHideDescription = Choose this option if you are not sure whether the two cases are the same or if you know that they are not. This will hide the case pair from the list. Please note that, when you refresh the view or come back to it later, it will re-appear as long as you don't choose a different time period in the creation date filter.
 infoHowToMergeCases = You can choose between two options when reviewing potentially duplicate cases:
-infoCalculateCompleteness = The <i>Calculate Completeness</i> button on top can be used to calculate the completeness values for all cases in the table. This is only necessary if there are cases in the list that do not yet have a completeness value. Normally, this value is automatically updated whenever the details of a case or one of its contacts or samples change.
-infoCaseCompleteness = Completeness is a measurement of how much vital data has already been entered into the case. This is composed of the following, weighted by their importance:<br/><ul><li>Has the case been classified?</li><li>Has the case been investigated?</li><li>Has at least one sample been taken?</li><li>Has at least one symptom been documented?</li><li>Has at least one contact been created?</li><li>Is there an outcome for the case?</li><li>Does the case have a birth date or age?</li><li>Does the case have a sex?</li><li>Has an onset date been specified?</li></ul>
+infoCaseCalculateCompleteness = The <i>Calculate Completeness</i> button on top can be used to calculate the completeness values for all cases in the table. This is only necessary if there are cases in the list that do not yet have a completeness value. Normally, this value is automatically updated whenever the details of a case or one of its contacts or samples change.
+infoCaseCompleteness = Completeness is a measurement of how much vital data has already been entered into the case. This is composed of the following, weighted by their importance:<br/><ul><li>For cases :</li><ul><li>Has the case been classified?</li><li>Has the case been investigated?</li><li>Has at least one sample been taken?</li><li>Has at least one symptom been documented?</li><li>Has at least one contact been created?</li><li>Is there an outcome for the case?</li><li>Does the case have a birth date or age?</li><li>Does the case have a sex?</li><li>Has an onset date been specified?</li></ul>
+infoHowToMergeContact = You can choose between two options when reviewing potentially duplicate contacts:
+infoContactCalculateCompleteness = The <i>Calculate Completeness</i> button on top can be used to calculate the completeness values for all contacts in the table. This is only necessary if there are contacts in the list that do not yet have a completeness value. Normally, this value is automatically updated whenever the details of a contact or one of its cases or samples change.
```

Comment:
Cases don't matter for contact completeness, visits however do; please update these strings with information that actually fits the logic that you've implemented